apps/glusterfs/app_volume.go

```diff
@@ -95,6 +95,18 @@ func (a *App) VolumeCreate(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	for _, volumeOption := range msg.Options {
+		switch volumeOption {
+		case "encryption":
```

Comment:
@obnoxxx This design is based on the discussion happend here https://github.com/heketi/heketi/issues/711. If we dont maintain a whitelist, heketi is not in control to validate the options or modify accordingly. There may be options which are conflicting each other. The intelligence of parsing it and validating those scenarios can be done in heketi if we follow this path. User may not know those kind of scenarios and they may pass all at same time. Thats why I thought we can have a kind of validation in heketi. But I am ready to hear other thoughts.